cmd/auto-updater/main.go

```diff
@@ -60,7 +61,7 @@ var (
 func init() {
 	rootCmd.AddCommand(runCmd)
 
-	runCmd.Flags().StringVar(&interval, "interval", "06h", "interval for checking the new version")
+	runCmd.Flags().StringVar(&interval, "interval", "06h", "interval for checking the new version, negative value will execute version check only once")
```

Comment:
This should now say "non-positive value" or "0 or less".

Reply:
ah yes :)